### PR TITLE
Indicator new tests

### PR DESCRIPTION
**Pull request by MarianMandziuk**
Provide new tests for indicators model and views

### CODE REVIEW

File: indicators/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                         pdi=0,
+                         idv=0,
+                         mas=32,
+                         uai=0,
+                         lto=89,
+                         ivr=43),
+        [0, 0, 32, 0, 89, 43],
+        "It is a very long na - LON",
+    ),
+)
```
Comment (falled10) [inline indicators/tests/test_models.py]: Can you move this code into test_models_data.py for example?

File: indicators/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+_search_data = (
+        ('bra', 1, ['BRA']),
+        ('an', 3, ['ALB', 'FRA', 'GER']),
+        ('ukr', 1, ['UKR']),
+        ('France', 1, ['FRA']),
+        ('Ukraine', 1, ['UKR']),
+        ('tuy', 0, []),
+        ('Frynce', 0, []),
+        (' fj 9', 0, [])
+    )
```
Comment (falled10) [inline indicators/tests/test_views.py]: Can you move this code into test_views_data.py for example?

File: indicators/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+@ddt
+class CountryIndicatorListViewTest(TestCase):
+
+    fixtures = ['indicators/tests_data/test_data.json']
+
+    @classmethod
+    def setUpTestData(cls):
+        role_name = 'Admin'
```
Comment (falled10) [inline indicators/tests/test_views.py]: Can you remove this magic string or replace it with ADMIN_ID from CultureAnalyzer.py?

File: indicators/tests/test_views.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                         msg=f'Incorrect order with search: {search_value}')
+
+
+class CountryIndicatorUpdateTest(TestCase):
+
+    fixtures = ['indicators/tests_data/test_data.json']
+
+    @classmethod
+    def setUpTestData(cls):
+        role_name = 'Admin'
```
Comment (falled10) [inline indicators/tests/test_views.py]: Can you remove this magic string or replace it with ADMIN_ID from CultureAnalyzer.py?

**Review by falled10 — CHANGES_REQUESTED** [review]

File: indicators/tests/test_views.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    @classmethod
+    def setUpTestData(cls):
+        permissions = ('view_countryindicator',)
+        create_user_with_role(ADMIN_ID, permissions)
+
+    def setUp(self):
+        self.client.login(username=USERNAME, password=PASSWORD)
+
+    def test_indicators_result_list_view(self):
+        response = self.client.get("/indicators/")
```
Comment (MrKreg) [inline indicators/tests/test_views.py]: On my opinion it would be better to use reverse method here.

File: indicators/tests/test_views.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.context['object_list']), 5)
+
+        response = self.client.get("/indicators/?page=2")
```
Comment (MrKreg) [inline indicators/tests/test_views.py]: On my opinion it would be better to use reverse method here.

File: indicators/tests/test_views.py [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.context['object_list']), 2)
+
+    @unpack
+    @data(*SEARCH_DATA)
+    def test_search_results(self, search_value, expected_count,
+                            expected_iso_code_list):
+        response = self.client.get(f"/indicators/?indicator_search"
```
Comment (MrKreg) [inline indicators/tests/test_views.py]: On my opinion it would be better to use reverse method here.

File: indicators/tests/test_views.py [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+    @classmethod
+    def setUpTestData(cls):
+        permissions = ('change_countryindicator', 'add_countryindicator')
+        create_user_with_role(ADMIN_ID, permissions)
+
+    def setUp(self):
+        self.client.login(username=USERNAME, password=PASSWORD)
+
+    def test_context_data(self):
+        response = self.client.get("/indicators/9/update/")
```
Comment (MrKreg) [inline indicators/tests/test_views.py]: On my opinion it would be better to use reverse method here.

File: indicators/tests/test_views.py [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(response.context['update'])
+
+        response = self.client.get("/indicators/create/")
```
Comment (MrKreg) [inline indicators/tests/test_views.py]: On my opinion it would be better to use reverse method here.

**Review by MrKreg — CHANGES_REQUESTED** [review]

**Review by falled10 — APPROVED** [review]

**Review by kolinkovlad — APPROVED** [review]

**Review by MrKreg — APPROVED** [review]

**Pull request merged**